### DISCUSSION TITLE
sveltekit user email checking

### DISCUSSION THREAD

**Discussion by Matrex7** [pocketbase/pocketbase#3395] [Q&A]
hello , i thought of verifying if someone is trying to register with an email of an already account , i saw some issues here and i found out its built in feature , but how do i inform the user trying to register that that email is already taken , i use sveltekit and here is some of my code ;
```
import { redirect ,fail,error,} from '@sveltejs/kit';
export const actions = {
    register:async   ({request,locals})=>{
        const body= Object.fromEntries(await  request.formData());        
        try {
            await locals.pb.collection("users").create(body);
            // @ts-ignore
           // await locals.pb.collection("users").requestVerification(body.email);
        } catch (error) {
          
            if (body.password != body.confirmpassword){
                return fail(422,{
                    email:body.email,
                    password:body.password,
                    confirmpassword:body.password,
                    error:"password must be the same as above"
                })
            }
            console.log(error)
        }
    }
}; 
```
how do i implement this stuff to show error to end user

**Comment by ganigeorgiev** [comment]
You can inspect the error response and more specifically `error.response.data`.

For more details about the available error fields you could check https://github.com/pocketbase/js-sdk#error-handling.

**Reply by ayamkv** [reply]
Well, i tried with this line of code

```js
catch (err) {
            console.log('Error : ', err);
            throw error(err.status, err.data);
        }
 ```
 
 but then it returns `data` as an empty object.
 
 ```
Object { code: 400, message: "Failed to authenticate.", data: {} }
​
code: 400
```

while in the pocketbase logs, returns : 
```json
{
  "errorDetails": "Invalid login credentials.",
  "errorMessage": "Failed to authenticate."
}
``` 
which i need the `errorDetails`
​
